code/datums/components/bloodysoles.dm

```diff
@@ -211,7 +211,6 @@
 	// Create new footprints
 	if(theoretical_new_blood >= BLOOD_FOOTPRINTS_MIN)
 		adjust_bloody_shoes(last_blood_state, blood_lost)
-		update_icon()
```

Comment:
Every step called human update icons :/ . Removing this seems to have no consequences on testing

Reply:
I feel like this should still call `update` when a threshold passes, ie if going back to 0 blood

Reply:
going back to 0 blood doesn't clear blood by default, not sure why update ever needs to be called